pages/common/q.md

```diff
@@ -0,0 +1,23 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file (with header for column name mapping):
+
+`q -d, -H "SELECT * from {{path/to/file}} where answer = 42"`
```

Comment:
Hey @waldyrious, thanks for the advice.

I've tried to simply the example by splitting it out.
I also reorganized the examples a bit to progress to the complex (imo) examples.
Please let me know if this is any better?